bundles/org.openhab.binding.bluelink/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -0,0 +1,493 @@
+			</channel>
+			<channel id="washer-fluid" typeId="warning">
+				<label>Washer Fluid Level Warning</label>
+			</channel>
+			<channel id="brake-fluid" typeId="warning">
+				<label>Brake Fluid Level Warning</label>
+			</channel>
+			<channel id="smart-key-battery" typeId="warning">
+				<label>Smart Key Battery Warning</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<!-- Channel Types -->
+
+	<channel-type id="locked">
+		<item-type>Switch</item-type>
+		<label>Locked</label>
+		<description>Vehicle door lock status</description>
+		<category>Lock</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>LockState</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="engine-running">
+		<item-type>Switch</item-type>
+		<label>Engine Running</label>
+		<description>Engine running status</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="battery-level">
+		<item-type>Number:Dimensionless</item-type>
+		<label>12V Battery Level</label>
+		<description>12V battery state of charge</description>
+		<category>BatteryLevel</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Level</tag>
+		</tags>
+		<state pattern="%.0f %%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="door-state">
+		<item-type>Contact</item-type>
+		<label>Door</label>
+		<description>Door open/closed state</description>
+		<category>Door</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="trunk-state">
+		<item-type>Contact</item-type>
+		<label>Trunk</label>
+		<description>Trunk open/closed state</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="hood-state">
+		<item-type>Contact</item-type>
+		<label>Hood</label>
+		<description>Hood open/closed state</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="window-state">
+		<item-type>Contact</item-type>
+		<label>Window</label>
+		<description>Window open/closed state</description>
+		<category>Window</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="hvac-on">
+		<item-type>Switch</item-type>
+		<label>Climate Control</label>
+		<description>Climate control active state</description>
+		<category>Climate</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Heating</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature-setpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Setpoint</label>
+		<description>Air temperature setpoint</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
As it is a setpoint, it is not a `Measurement`, but the desired temperature. So it would be `Control`. But it is also readonly, so i would say `Status`